stripe_official.php

```diff
@@ -1663,7 +1664,7 @@ public function hookActionOrderStatusUpdate($params)
     /**
      * Load JS on the front office order page
      */
-    public function hookHeader()
+    public function hookDisplayHeader()
```

Comment:
are you sure for 1.6.0, you don't need an upgrade ?

Reply:
hook displayHeader is full compatible with ps 1.6